memtestG80_core.cu

```diff
@@ -62,7 +62,7 @@ uint memtestState::allocate(uint mbToTest) {
 		if (megsToTest == 0) return 0;
 		
 		try {
-			if (cudaMalloc((void**)&devTestMem,megsToTest*1048576UL) != cudaSuccess) throw 1;
+			if (cudaMalloc((void**)&devTestMem,megsToTest*1048576ULL) != cudaSuccess) throw 1;

```

Comment:
Thanks! I think you're right that this was problematic. Looking more closely at the spec for `cudaMalloc`, what do you think of `((size_t) megsToTest) * 1048576` instead?

Reply:
(that is, for platform independence, this should actually be `size_t`, not `unsigned long long`)

Reply:
I agree, that’s more correct. I’ll fix this in a bit.

Reply:
fixed this